packages/neo4j/package.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "@next-auth/neo4j-adapter",
+  "version": "0.1.0",
+  "description": "neo4j adapter for next-auth.",
+  "homepage": "https://next-auth.js.org",
+  "repository": "https://github.com/nextauthjs/adapters",
+  "bugs": {
+    "url": "https://github.com/nextauthjs/next-auth/issues"
+  },
+  "author": "Davey Brown",
+  "main": "dist/index.js",
+  "license": "ISC",
+  "keywords": [
+    "next-auth",
+    "next.js",
+    "oauth",
+    "neo4j"
+  ],
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "test:watch": "./tests/test.sh -w",
+    "test": "yarn build && ./tests/test.sh",
```

```suggestion
    "test": "./tests/test.sh",
```

Build is already run in the pipeline, no need here.